bigquery/storage/managedwriter/integration_test.go

```diff
@@ -894,8 +894,16 @@ func testInstrumentation(ctx context.Context, t *testing.T, mwClient *Client, bq
 			t.Errorf("%q: expected 1 row of metrics, got %d", tv.Name, mlen)
 			continue
 		}
-		if len(metricData[0].Tags) != 1 {
-			t.Errorf("%q: only expected 1 tag, got %d", tv.Name, len(metricData[0].Tags))
+		var wantTags int
+		switch tv.Name {
+		case "cloud.google.com/go/bigquery/storage/managedwriter/stream_open_count",
+			"cloud.google.com/go/bigquery/storage/managedwriter/stream_open_retry_count":
+			wantTags = 0
+		default:
+			wantTags = 1
+		}
```

Comment:
I feel like this check for the amount of tags ended up a bit confusing. Maybe instead of iterating on `testedViews`, a custom struct can be created that have a `View` and expected tags, following a table driven pattern.

Reply:
added a wantTags map for validating this more explicityly.